testing/tests/DevExpress.ui.widgets/speedDialAction.tests.js

```diff
@@ -388,4 +389,59 @@ QUnit.module("add visible option", (hooks) => {
     });
 });
 
+QUnit.module("add label option", (hooks) => {
+    let firstSDA;
+    let secondSDA;
+    hooks.afterEach(() => {
+        firstSDA.dispose();
+        secondSDA.dispose();
+    }),
+    test("check rendering", (assert) => {
+        config({
+            floatingActionButtonConfig: {
+                position: {
+                    at: "right bottom",
+                    my: "right bottom",
+                    offset: "-16 -16"
+                }
+            }
+        });
+
+        firstSDA = $("#fab-one").dxSpeedDialAction({
+            label: "first action"
+        }).dxSpeedDialAction("instance");
+
+        assert.equal($(FAB_MAIN_SELECTOR).find(FAB_LABEL_SELECTOR).text(), "first action", "FAB has label");
+        assert.ok($(FAB_MAIN_SELECTOR).hasClass("dx-fa-button-has-text"), "FAB has class");
+
+        let $fabMainContent = $(FAB_MAIN_SELECTOR).find(".dx-overlay-content");
+
+        assert.equal($fabMainContent.offset().top, $(window).height() - ($fabMainContent.outerHeight() + 16), "default position top");
```

Comment:
I think we need a named variable for $(window).height() - ($fabMainContent.outerHeight() + 16) and better comment for this assert to understand what this assert is doing.